sdk/sdk-api/src/main/java/mil/emp3/api/utils/MilStdUtilities.java

```diff
@@ -46,4 +54,164 @@ public static String geoAltitudeModeToString(IGeoAltitudeMode.AltitudeMode mode)
         }
         return "";
     }
+
+    public static String getMilStdSinglePointIconURL(final MilStdSymbol feature,
+            MilStdLabelSettingEnum eLabelSetting, java.util.Set<IGeoMilSymbol.Modifier> labelSet,
+            int iIconSize, boolean selected, IGeoColor selectedStrokeColor, IGeoColor selectedTextColor) {
+        int iKey;
+        String value;
+        String UniqueDesignator1 = null;
+        String params = "";
+        java.util.HashMap<IGeoMilSymbol.Modifier, String> geoModifiers = feature.getModifiers();
+        SparseArray<String> attributes = feature.getAttributes(iIconSize,
+                selected, selectedStrokeColor, selectedTextColor);
+
+        if (null != attributes) {
```

Comment:
Will this method still produce meaningful output if the following conditions are all FALSE?

> if (null != attributes)
> if ((geoModifiers != null) && !geoModifiers.isEmpty())
> if ((feature.getName() != null) && !feature.getName().isEmpty())